build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.CS.targets

```diff
@@ -1,23 +1,23 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapAutoInitialize)'==''">
-        <WindowsAppSDKBootstrapAutoInitialize>true</WindowsAppSDKBootstrapAutoInitialize>
+    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapInitialize)'==''">
```

Comment:
this is unnecessary.

https://github.com/microsoft/WindowsAppSDK/blob/48e6b8f1766210977b9c32f1b35641ae71dce551/build/NuSpecs/Microsoft.WindowsAppSDK.Foundation.targets#L6:
  <Import Project="$(MSBuildThisFileDirectory)Microsoft.WindowsAppSDK.Bootstrap.CS.targets" Condition="'$(WindowsAppSdkBootstrapInitialize)' == 'true'"/>

Reply:
Because `build\NuSpecs\Microsoft.WindowsAppSDK.MddCommon.targets` handles it?

I overlooked that file as it didn't mention bootstrap. Mdd=DynamicDependencies. Generic facility for any package. Bootstrap=DynDep-specialization for WinAppSDK's runtime. 

Any objection to renaming `...MddCommon...` to `...BootstrapCommon...` ?